### PR TITLE
Version 3.1 beta 01

### PR DESCRIPTION
**Pull request by AnthonySteele**
_Summarise the changes this Pull Request makes._

version metadata so that current changes can be released as a beta for test

### CODE REVIEW

**Comment by AnthonySteele** [comment]
So what should `SetAppVeyorBuildVersion.ps1` change to? Is there an example? You don't want a `-something` suffix on every tagged build, as many of them will not be betas. 

I have checked a few other repos (NLog.StructuredLogging.Json, JustSaying) and they do the same as this.

I'm happy for this to be improved some time, but I don't know what to, and want to get code out.

**Comment by martincostello** [comment]
Pseudo code:

```ps
else {
    if ($APPVEYOR_REPO_TAG_NAME.Contains("-")) {
        $versionSuffix = $APPVEYOR_REPO_TAG_NAME.Substring($APPVEYOR_REPO_TAG_NAME.IndexOf("-") +1)
    }
}
```

**Comment by martincostello** [comment]
This doesn't hold you trying it out either, as the AppVeyor build already publishes every build to a pseudo-private NuGet feed you can use internally which you can get the URL of from the NuGet tab in the build's settings over in AppVeyor.

**Comment by AnthonySteele** [comment]
The Pseudocode appears to be correct.

**Comment by martincostello** [comment]
Looks like I missed prefixing the values with `$env:`.

**Comment by martincostello** [comment]
So: `$env:APPVEYOR_REPO_TAG_NAME`

File: version.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.1.0</VersionPrefix>
-    <VersionSuffix></VersionSuffix>
+    <VersionSuffix>beta-01</VersionSuffix>
```
Comment (martincostello) [inline version.props]: Just need to re-blank this now.

**Comment by AnthonySteele** [comment]
The recommendation is to start tags with a `v`, so it should be `v3.1.0-beta01` - it's not consistent, but that is the prefered one.

**Pull request merged**